mobilizon_bots/models/notification.py

```diff
@@ -0,0 +1,22 @@
+from tortoise import fields
+from tortoise.models import Model
+
+
+class Notification(Model):
+    id = fields.UUIDField(pk=True)
+    # TODO: Make actual IntEnumField
+    status = fields.IntField()
```

Comment:
I think this one should be addressed in this PR. We saw it's part of Tortoise anyway, no?

Reply:
Yes, I also wanted to discuss the Notification model synchronously as last time we didn't state clearly the scope/requirements of the model. Anyway I'll try to set up a proposal before our next meeting.